MonetClient/Classes/Api/SortOrderDirection.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  SortOrder.swift
+//  MonetClient
+//
+//  Created by 伊藤拓海 on 2019/12/10.
+//  Copyright © 2019 monet-dev. All rights reserved.
+//
+
+import Foundation
+
+/// Sort 順序
+enum SortOrderDirection: String, Encodable {
+    case asc = "ASC"
+    case desc = "DESC"
```

Comment:
Enumに値を持たせるのに、`rawValue`よりも変数で値を定義する方が汎用性が高いことに気づいたので、そっちの方がいいかと思う。
```suggestion
   var code: String {
       switch self {
       case .asc: return "ASC"
       case .desc: return "DESC"
   }
```

Reply:
encodable にしたいので、 request のみこの形式でいくことで決定。
entity とかそれ以外の場所では変数で持つことにしましょう！